ansible_bender/constants.py

```diff
@@ -32,3 +32,30 @@
       - another_package
       state: present
 """
+KEYS = [
+        # optional keys 
+        "build_id",
+        "build_finished_time",
+        "build_start_time",
+        "build_volumes",
+        "layers",
+        "final_layer_id",
+        "layer_index",
+        "build_container",
+        "cache_tasks",
+        "log_lines",
+        "layering",
+        "squash",
+        "debug",
+        "verbose",
+        "pulled",
+        "ansible_extra_args",
+        "python_interpreter",
+        # required keys
+        "playbook_path",
+        "metadata",
+        "state",
+        "base_image",
+        "target_image",
+        "builder_name"
+]
```

Comment:
these are the keys from the database, not vars -> ansible_bender

Reply:
@TomasTomecek Sorry! I will work on it